core/validation/initial_check.py

```diff
@@ -120,6 +125,20 @@ def pre_transformation_check(submission_details: dict[str, dict[str, dict]]) ->
         if failure := check_values(value_descriptor, entered_value, expected_values):
             failures.append(failure)
 
+    # we do not want to return 'Place Name vs Fund Type' failure if either are incorrect
+    failures = [
+        failure
+        for failure in failures
+        if not (
+            failure.value_descriptor == "Place Name vs Fund Type"
+            and any(
+                other_failure.value_descriptor in ["Fund Type", "Place Name"]
+                for other_failure in failures
+                if other_failure != failure
+            )
+        )
+    ]
```

Comment:
I think the other_failure != failure clause is unnecessary. It would work the same as
```
    failures = [
        failure
        for failure in failures
        if not (
            failure.value_descriptor == "Place Name vs Fund Type"
            and any(
                other_failure .value_descriptor in ["Fund Type", "Place Name"]
                for other_failure in failures
            )
        )
    ]
```
I also think there might be a more succinct refactor here as well

Reply:
we know that failure.value_descriptor == "Place Name vs Fund Type" so it dosen't == "Fund Type" or "Place Name"